### PR TITLE
photoshoot hints

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/bronze/usaco-1227.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+using std::cout;
+using std::endl;
+using std::string;
+
+int main() {
+	int cow_num;
+	string cows;
+	std::cin >> cow_num >> cows;
+	assert(cows.size() == cow_num && cow_num % 2 == 0);
```
Comment (envyaims) [inline solutions/bronze/usaco-1227.mdx:52]: whats the point of this

**Review by envyaims — APPROVED** [review]
afsaefsd

File: solutions/bronze/usaco-1227.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/bronze/usaco-1227.mdx:52]: i let you keep your `n`, you let me keep this

**Review by ChuangSheep — APPROVED** [review]
<del>streams are meant to be closed</del>
it's alright

**Pull request merged**